Permanent/ViewControllers/SideMenuViewController.swift

```diff
@@ -82,14 +88,23 @@ class SideMenuViewController: BaseViewController<AuthViewModel> {
     }
     
     func updateLeftSideMenu() {
+        var menuIndexPaths: [IndexPath] = []
+        var menuTitles: [DrawerOption] = []
+        if let selectedArchivePermission = AuthenticationManager.shared.session?.selectedArchive?.permissions(), selectedArchivePermission.contains(.legacyPlanning) {
```

Comment:
This logic should happen in viewModel. The viewController should just call a method like viewModel.hasLegacyPermissions()

Reply:
I will add this to viewModel.